python/kfserving/docs/KFServingClient.md

```diff
@@ -16,11 +16,82 @@ The APIs for KFServingClient are as following:
 
 Class | Method |  Description
 ------------ | ------------- | -------------
+KFServingClient | [create_credentials](#create_credentials) | Create Credentials|
 KFServingClient | [create](#create) | Create KFService|
 KFServingClient | [get](#get)    | Get the specified KFService|
 KFServingClient | [patch](#patch)  | Patch the specified KFService|
 KFServingClient | [delete](#delete) | Delete the specified KFService |
 
+## create_credentials
+> create_credentials(storage_type, namespace=None, **kwargs)
+
+Create or update `Secret` and `Service Account` for GCS and S3 according to credentials information. Once the `Service Account` created or updated, user can configure the `Service Account` in the [V1alpha1ModelSpec](V1alpha1ModelSpec.md) for kfserving.
+
+The API returns name of the `Service Account`.
+
+### Example
+
+Example for creating GCP credentials.
+```python
+from kfserving import KFServingClient
+
+KFServing = KFServingClient()
+KFServing.create_credentials(storage_type='GCS',
+                             namespace='kubeflow',
+                             credentials_file='/tmp/gcp.json')
+```
+
+Example for creating AWS credentials.
+```python
+from kfserving import KFServingClient
+
+KFServing = KFServingClient()
+KFServing.create_credentials(storage_type='S3',
+                             namespace='kubeflow',
+                             credentials_file='/tmp/aws/credentials',
+                             s3_profile='default',
+                             s3_endpoint='s3.us-west-2.amazonaws.com',
+                             s3_region='us-west-2',
+                             s3_use_https='1',
+                             s3_verify_ssl='0')
+```
+
+The created `Secret` and `Service Account` will be shown as following:
+```
+INFO:kfserving.api.set_credentials:Created Secret: kfserving-secret-6tv6l in namespace kubeflow
+INFO:kfserving.api.set_credentials:Created Service account: kfserving-sa-tj444 in namespace kubeflow
```

Comment:
Can we name this something more deterministic? "kfserving-service-credentials"

Reply:
OK, Defined the default service account name to "kfserving-service-credentials".